bcrs-ui/src/store/state/resource-model.ts

```diff
@@ -0,0 +1,14 @@
+import { ResourceExampleIF } from '@/interfaces'
+
+export const resourceModel: Array<ResourceExampleIF> = [
```

Comment:
I don't understand how this array works. Comments please?

Reply:
So in this instance, it's an Array of `ResourceExamples`. Applying the interface to the type of objects we want to see in that array so the resourceModel must conform to the interface, ie an array of our specified objects.     This was done purely as an example for the resource files, just to demonstrate setting a resource file whether it be JSON or an Array or an object and so on. I just randomly settled on using an Array 😄  Would you like a comment in the code as well?

Reply:
Yes, please add a short code comment. I'm wondering how to use this resource and when I would see message 1 vs message 2?

Reply:
Okie doke, no problem at all, thanks!